packages/twenty-front/src/hooks/usePageChangeEffectNavigateLocation.ts

```diff
@@ -19,16 +19,17 @@ export const usePageChangeEffectNavigateLocation = () => {
   const isWorkspaceSuspended = useIsWorkspaceActivationStatusEqualsTo(
     WorkspaceActivationStatus.SUSPENDED,
   );
+
   const { defaultHomePagePath } = useDefaultHomePagePath();
 
   const isMatchingOpenRoute =
     isMatchingLocation(AppPath.Invite) ||
-    isMatchingLocation(AppPath.ResetPassword) ||
-    isMatchingLocation(AppPath.VerifyEmail);
+    isMatchingLocation(AppPath.ResetPassword);
 
   const isMatchingOngoingUserCreationRoute =
```

Comment:
Why move VerifyEmail here?

Reply:
VerifyEmail should work like Verify and avoid skipping the redirection at line 50.